locations/spiders/fjallraven.py

```diff
@@ -7,10 +7,12 @@ class FjallravenSpider(Where2GetItSpider):
     item_attributes = {
         "brand": "Fjällräven",
         "brand_wikidata": "Q1422481",
-        "extras": Categories.SHOP_CLOTHES.value,
+        "extras": Categories.SHOP_OUTDOOR.value,
```

Comment:
This is one of the issues with hardcoding categories, if we had simply no applied one, we would have gotten it from NSI. If NSI changed, we'd have gotten it automatically.
```suggestion
```